web_m2x_options/static/src/js/form.js

```diff
@@ -479,4 +475,21 @@ odoo.define("web_m2x_options.web_m2x_options", function (require) {
             }
         },
     });
+
+    // Extending class to allow change the limit of o2m registry entries using the
+    // system parameter "web_m2x_options.field_limit_entries".
+    FormView.include({
+        _setSubViewLimit: function (attrs) {
+            this._super(attrs);
+            var limit = 0;
+            ir_options.forEach((element) => {
+                if (element.key == "web_m2x_options.field_limit_entries") {
+                    limit = element.value;
+                }
+            });
```

Comment:
May I know why this is different than original? https://github.com/OCA/web/pull/1728/commits/a9c15b35ec54ab7ca2da3f75e413bb553690e8c7#diff-d19550cf74a7a0c677bde2f3bf36f00cabf3b3e8b2518402ce2fd53e7e329b87R370

Reply:
Hi @hailangvn , `ir_options["web_m2x_options.field_limit_entries"]` return `Array`. Each value in `Array` is an `Object` so I have to use loop to get `element.value`. Do you have a better idea?
![image](https://user-images.githubusercontent.com/61400218/146628848-8a06136f-4647-45c8-8b4d-0b658946dcbd.png)